package.json

```diff
@@ -10,7 +10,8 @@
   },
   "scripts": {
     "start": "parallelshell 'bundle exec jekyll serve --watch --baseurl ''' 'gulp watch'",
-    "build": "gulp build"
+    "build": "gulp build",
+    "cf-link": "npm link cf-pagination"
```

Comment:
See [comment in corresponding PR](https://github.com/cfpb/capital-framework/pull/326#discussion_r72326259)